src/components/Table.svelte

```diff
@@ -22,17 +22,11 @@
     let currentRow = undefined
     let extraContent = undefined
 
-    if (e.target.parentNode.classList.contains("title")) {
-      title = e.target.parentNode
-      currentRow = title.nextElementSibling
-      extraContent = e.target.parentNode.nextElementSibling
-    } else {
-      title = e.target.parentNode.parentNode
-      currentRow = title.nextElementSibling
-      extraContent = e.target.parentNode.parentNode.nextElementSibling
-    }
+    title = e.target.parentNode.parentNode
```

Comment:
The code needs the if statement to check from where the click is coming. If you click outside the title div the click is not working and no showing the hidden row.
```suggestion
    if (e.target.parentNode.classList.contains('title') ) {
      title = e.target.parentNode
      currentRow = title.nextElementSibling
      extraContent = e.target.parentNode.nextElementSibling;
    } else {
      title = e.target.parentNode.parentNode
      currentRow = title.nextElementSibling
      extraContent = e.target.parentNode.parentNode.nextElementSibling;
    }
```
![image](https://github.com/CSIS-iLab/poni-table/assets/51330062/f9fd67bc-1b45-4b33-90ca-934f1272eccd)

Reply:
Gracias for catching that. I remember changing this because something I needed wasn't working but I don't remember what it was now. I made this change and everything looks good 👍